src/test/java/seedu/mark/logic/commands/CommandTestUtil.java

```diff
@@ -120,4 +127,48 @@ public static void showBookmarkAtIndex(Model model, Index targetIndex) {
         assertEquals(1, model.getFilteredBookmarkList().size());
     }
 
+    /**
+     * A default Storage Stub that has all of its methods failing.
+     */
+    public static final class StorageStub implements Storage {
```

Comment:
How about putting this in a separate file?

Reply:
Right, that makes more sense. Will do.